wordpress-mobile/WordPress-iOS
# QS v2: Quick Start Section in Blog Details

Refs #10860

This adds the Quick Start section to the blog details, and updates the cells to match the mockups.

Example | Example with one complete
-----|-----
![simulator screen shot - iphone xr - 2019-01-25 at 16 11 02](https://user-images.githubusercontent.com/517257/51779864-de7b2c00-20be-11e9-8f82-ebd4fd752eb4.png) | ![simulator screen shot - iphone xr - 2019-01-25 at 16 34 26](https://user-images.githubusercontent.com/517257/51779896-18e4c900-20bf-11e9-8a2c-6ea2e726d188.png)

To test:
- put `return true` in `shouldShowQuickStartChecklist()`
- ensure that the blog details view matches the mockups

Update release notes:
- These changes won't be visible to users until the feature flag is removed
